debian/cloudstack-agent.install

```diff
@@ -18,12 +18,11 @@
 /etc/cloudstack/agent/agent.properties
 /etc/cloudstack/agent/environment.properties
 /etc/cloudstack/agent/log4j-cloud.xml
+/etc/default/cloudstack-agent
 /etc/profile.d/cloudstack-agent-profile.sh
 /etc/logrotate.d/cloudstack-agent
-/etc/init.d/cloudstack-agent
 /usr/bin/cloudstack-setup-agent
```

Comment:
Maybe, never tried that before. Might be possible. But shall we do that in a different PR? Merge this one and then fix that.